### PR TITLE
make mc ls output consistent while listing

### PR DESCRIPTION
**Pull request by poornas**
directories on file system.
```
➜  mc git:(master) mc ls ./vendor/
[2019-03-21 12:14:57 PDT] 4.0KiB vendor/github.com/
[2019-03-21 12:14:57 PDT] 4.0KiB vendor/go.uber.org/
[2019-03-11 11:26:59 PDT] 4.0KiB vendor/golang.org/
[2019-03-21 12:14:58 PDT] 4.0KiB vendor/google.golang.org/
[2019-03-21 12:14:58 PDT] 4.0KiB vendor/gopkg.in/
[2019-03-21 12:59:55 PDT] 6.8KiB vendor/modules.txt
```
```
➜  mc git:(master) mc ls vendor/  
[2019-03-21 12:14:57 PDT] 4.0KiB github.com/
[2019-03-21 12:14:57 PDT] 4.0KiB go.uber.org/
[2019-03-11 11:26:59 PDT] 4.0KiB golang.org/
[2019-03-21 12:14:58 PDT] 4.0KiB google.golang.org/
[2019-03-21 12:14:58 PDT] 4.0KiB gopkg.in/
[2019-03-21 12:59:55 PDT] 6.8KiB modules.txt
```
Both commands should give identical listing , just like unix ls command

### CODE REVIEW

File: cmd/ls.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```
Comment (kannappanr) [inline cmd/ls.go]: why is this change needed?

**Comment by kannappanr** [comment]
Tested. LGTM. Just need clarification on another change that was made.

File: cmd/ls.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/ls.go]: That happens automatically I think due to goimports

Comment (poornas) [reply cmd/ls.go]: there is a reference to humanize.IBytes to humanize size on line 50,  this was automatically added by go tools installed on the editor.
